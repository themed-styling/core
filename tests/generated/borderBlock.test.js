import 'jest-styled-components'

import { borderBlock } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(borderBlock, 'borderBlock')
