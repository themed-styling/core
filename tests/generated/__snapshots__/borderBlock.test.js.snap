// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`borderBlock renders 0 1`] = `
Array [
  .c0 {
  border-block: 0;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: 0;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: 0;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlock renders function 1`] = `
Array [
  .c0 {
  border-block: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: has;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: has;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlock renders invalid theme path 1`] = `
Array [
  .c0 {
  border-block: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlock renders negative number 1`] = `
Array [
  .c0 {
  border-block: -54881px;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: -54881px;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: -42365px;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: -42365px;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlock renders object [breakpoints + standard] 1`] = `
Array [
  .c0 {
  border-block: July;
}

@media screen and (min-width:November) {
  .c0 {
    border-block: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  border-block: July;
}

@media screen and (min-width:November) {
  .c0 {
    border-block: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  border-block: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-block: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block: June;
  }
}

<div
    className="c0"
  />,
  .c0 {
  border-block: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-block: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlock renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    border-block: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    border-block: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-block: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-block: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlock renders object [mediaQueries + standart] 1`] = `
Array [
  .c0 {
  border-block: July;
}

@media May {
  .c0 {
    border-block: August;
  }
}

@media April {
  .c0 {
    border-block: September;
  }
}

@media January {
  .c0 {
    border-block: November;
  }
}

@media April {
  .c0 {
    border-block: August;
  }
}

@media June {
  .c0 {
    border-block: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  border-block: July;
}

@media May {
  .c0 {
    border-block: August;
  }
}

@media April {
  .c0 {
    border-block: September;
  }
}

@media January {
  .c0 {
    border-block: November;
  }
}

@media April {
  .c0 {
    border-block: August;
  }
}

@media June {
  .c0 {
    border-block: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  border-block: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-block: November;
  }
}

@media April {
  .c0 {
    border-block: January;
  }
}

@media January {
  .c0 {
    border-block: December;
  }
}

@media April {
  .c0 {
    border-block: April;
  }
}

@media June {
  .c0 {
    border-block: May;
  }
}

<div
    className="c0"
  />,
  .c0 {
  border-block: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-block: November;
  }
}

@media April {
  .c0 {
    border-block: January;
  }
}

@media January {
  .c0 {
    border-block: December;
  }
}

@media April {
  .c0 {
    border-block: April;
  }
}

@media June {
  .c0 {
    border-block: May;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlock renders object [mediaQueries] 1`] = `
Array [
  @media May {
  .c0 {
    border-block: July;
  }
}

@media April {
  .c0 {
    border-block: August;
  }
}

@media January {
  .c0 {
    border-block: September;
  }
}

@media April {
  .c0 {
    border-block: November;
  }
}

@media June {
  .c0 {
    border-block: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    border-block: July;
  }
}

@media April {
  .c0 {
    border-block: August;
  }
}

@media January {
  .c0 {
    border-block: September;
  }
}

@media April {
  .c0 {
    border-block: November;
  }
}

@media June {
  .c0 {
    border-block: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-block: January;
  }
}

@media April {
  .c0 {
    border-block: December;
  }
}

@media January {
  .c0 {
    border-block: April;
  }
}

@media April {
  .c0 {
    border-block: May;
  }
}

@media June {
  .c0 {
    border-block: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-block: January;
  }
}

@media April {
  .c0 {
    border-block: December;
  }
}

@media January {
  .c0 {
    border-block: April;
  }
}

@media April {
  .c0 {
    border-block: May;
  }
}

@media June {
  .c0 {
    border-block: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlock renders postitive number 1`] = `
Array [
  .c0 {
  border-block: 54881px;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: 54881px;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: 42365px;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: 42365px;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlock renders string 1`] = `
Array [
  .c0 {
  border-block: July;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: July;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: June;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: June;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlock renders undefined 1`] = `
Array [
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
]
`;

exports[`borderBlock renders valid theme path 1`] = `
Array [
  .c0 {
  border-block: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-block: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
